spec/index.html

```diff
@@ -5669,248 +5731,213 @@ <h5>NOT IN</h5>
+            </pre>
+
+            <p>This function cannot be used directly in expressions. The purpose
+              of this function is to define the semantics of the "=" operator when applied to
+              two RDF terms that do not fall into any of the other, more concrete cases
+              covered in the operator mapping table in Section
+              <a href="#OperatorMapping" class="sectionRef"></a>.
+            </p>
+
+            <p>The result of this function is determined by going through the following steps.</p>
+
+            <ol>
+              <li>If <code>term1</code> and <code>term2</code> are
+                <a data-cite="RDF12-CONCEPTS#dfn-rdf-term-equality">equal RDF terms</a>,
+                then return TRUE.
+              </li>
+              <li>If <code>term1</code> and <code>term2</code> are both
+                <a data-cite="RDF12-CONCEPTS#dfn-literal">literals</a>
+                and one or both of these literals are known to be
+                <a data-cite="RDF12-CONCEPTS#dfn-ill-typed">ill-typed</a>,
+                then produce a type error.
               </li>
               <li>
-                <span class="name">term1</span> is a <span class="literal type">literal</span> and
-                <span class="name">term2</span> is a <span class="literal type">literal</span> such that
-                these two <span class="literal type">literals</span> are equal as per the notion of <a data-cite="RDF12-CONCEPTS#dfn-literal-term-equality">Literal term equality</a> of
-                [[RDF12-CONCEPTS]].
+                "NaN"^^xsd:double and "NaN"^^xsd:float are considered to 
+                represent the same value.
+                If <code>term1</code> and <code>term2</code> are
+                both "NaN" for either xsd:double or xsd:float then 
+                return TRUE.
               </li>
               <li>
-                <span class="name">term1</span> is a <span class="literal type">literal</span> and
-                <span class="name">term2</span> is a <span class="literal type">literal</span>
-                such that the <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>
-                of each of these two literals is in the set of
-                <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>
-                and both literals have the same <a data-cite="RDF12-CONCEPTS#dfn-literal-value">literal value</a>.
+                For xsd:double and xsd:float, `+0`, `-0` and `0` are same value.
+                If <code>term1</code> and <code>term2</code> each
+                have datatype IRI xsd:double or xsd:float and
+                a lexical form of one of these zeros, then return TRUE.
+              </li>
```

Comment:
Is this step useful? If I am not mistaken `-0` and `+0` are equal according to `op:numeric-equal` <q>Equality is identity, except that  0 = −0  (although they are not identical) and  NaN ≠ NaN  (although NaN is of course identical to itself).</q> [XSD datatypes](https://www.w3.org/TR/xmlschema11-2/#double)

Reply:
I think it worth calling out this case for clarity (mild advocacy).

(I think this is in an up-PR comment but the GH interface for long PRs makes finding things difficult.)

Reply:
Yes!

Reply:
It is now a note.

Reply:
thanks!